account_asset_depr_line_cancel/models/account_asset_asset.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015-2017 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import models, api
+
+
+class AccountAssetDepreciationLine(models.Model):
+    _inherit = 'account.asset.depreciation.line'
+
+    @api.multi
+    def unlink_move(self):
+        moves = self.mapped('move_id')
+        moves.filtered(lambda x: x.state == 'posted').button_cancel()
```

Comment:
Well, in Spain we make a renumbering at the end of the year, so that's not an important question, but if there are problems with this in other countries, we can put it on the README so that we advise not to install it there.